api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -2128,12 +2126,87 @@ private void enableHIV() {
     }
 
     private void enableHIVProgram() {
-        configureBasicProgramDashboard(PihEmrConfigConstants.PROGRAM_HIV_UUID);
 
-        // additional columns to add to the HIV Program Dashboard
+        String programUuid = PihEmrConfigConstants.PROGRAM_HIV_UUID;
+
+        int firstColumnIndex = 0;
+        int secondColumnIndex = 0;
```

Comment:
Did you see that the order of widgets was previously handled via constants in the CALC (lines 589 and above..)  

I do think this in-line style is cleaner, but let's remove the above constants if they aren't still in use?

Reply:
I've remove the constants, thanks @mogoodrich